test/controllers/FileChecksResultsControllerSpec.scala

```diff
@@ -148,8 +148,16 @@ class FileChecksResultsControllerSpec extends FrontEndTestHelper {
         mockGraphqlResponse(userType, fileStatusResponse, filePathResponse)
```

Comment:
These tests no longer reflect what the `FileChecksResultsController` is doing so should be refactored:

- Separate out the `"render the $userType fileChecksResults page with the confirmation box"` into separate tests for standard and judgment users outside of the wrapper: `forAll(userTypes) { userType => ...`
- Have the following new tests:
`"FileChecksResultsController GET" should render the fileChecksResults page with the confirmation box for a standard user`
`"FileChecksResultsController GET" should render the fileChecksResults page with the confirmation box for a judgment user when the file checks fail`
`"FileChecksResultsController GET" should render the transfer complete page for a judgment user when the file checks succeed`